File: diet/src/main/java/com/example/diet/Domain/WeightRecord.java
package com.example.diet.Domain;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class WeightRecord {
    private long usr_id;
    private float usr_weight;
    private String date;

}

File: frontend/DayDiet_google/app/src/main/java/com/example/daydiet/model/Domain/WeightRecord.kt
package com.example.daydiet.model.Domain

import com.example.daydiet.model.entity.BaseResponse

data class WeightRecord (
    val usr_id : Long,
    val usr_weight : Float,
    val date : String
)
// 获取用户的体重记录
data class WeightResponse(var data:List<WeightRecord> ): BaseResponse() {}